packages/anoncreds/src/models/internal.ts

```diff
@@ -9,23 +9,32 @@ export interface CredentialInfo {
   credentialRevocationId?: string | undefined
 }
 
-export interface RequestedAttribute {
+export interface AnonCredsRequestedAttribute {
   credentialId: string
   timestamp?: number
   revealed: boolean
-  credentialInfo: CredentialInfo
+  credentialInfo: AnonCredsCredentialInfo
   revoked?: boolean
 }
 
-export interface RequestedPredicate {
+export interface AnonCredsRequestedPredicate {
   credentialId: string
   timestamp?: number
-  credentialInfo: CredentialInfo
+  credentialInfo: AnonCredsCredentialInfo
   revoked?: boolean
 }
 
-export interface RequestedCredentials {
-  requestedAttributes?: Record<string, RequestedAttribute>
-  requestedPredicates?: Record<string, RequestedPredicate>
+export interface AnonCredsRequestedCredentials {
+  requestedAttributes?: Record<string, AnonCredsRequestedAttribute>
+  requestedPredicates?: Record<string, AnonCredsRequestedPredicate>
   selfAttestedAttributes: Record<string, string>
 }
+
+export interface AnonCredsCredentialRequestMetadata {
+  master_secret_blinding_data: {
+    v_prime: string
+    vr_prime: string | null
```

Comment:
Is there a reason for the `null` instead of `vr_prime?`?

Reply:
It is present with a `null` value by indy sdk itself. So it's more correct do to `vr_prime: string | null`